features/fakes/FakeIdBrokerClient.php

```diff
@@ -84,7 +85,7 @@ public function updateUserLastLogin(string $employeeId): array
      * @return array|null
      * @throws Exception
      */
-    public function mfaCreate($employee_id, $type, $label = null)
+    public function mfaCreate($employee_id, $type, $label = null): ?array
```

Comment:
no strong typing on the parameter list? I realize that sometimes uglinesses exist and not typing is nicer. Just asking, because you strong typed below.

Reply:
Not sure how I missed those, thanks.